sw/airborne/mcu_periph/i2c.h

```diff
@@ -33,6 +33,7 @@
 #include "std.h"
 
 #include "mcu_periph/i2c_arch.h"
+#include "mcu_periph/softi2c.h"
```

Comment:
is it really necessary to include this ?

Reply:
It was there so that all modules currently including i2c.h can also use the softi2c peripherals. I have now added the required extern declarations and removed the include, which works just as well.